.eslintrc

```diff
@@ -1,8 +1,24 @@
 {
   "extends": ["eslint:recommended"],
+
   "env": {
-    "es6": true,
     "browser": true,
+    "es6": true,
     "node": true
+  },
+
+  "rules": {
+    "curly": 0,
+    "eol-last": 1,
+    "eqeqeq": 2,
+    "indent": [2, 2],
+    "no-throw-literal": 2,
+    "no-unused-vars": [2, {"vars": "all", "args": "none"}],
+    "no-var": 0,
```

Comment:
Disabled the `curly` and `no-var` rules here, since they were generating a bit of noise.

- with `"curly": 1`: ✖ 3 problems (0 errors, 3 warnings)
- with `"no-var": 1`: ✖ 22 problems (0 errors, 22 warnings)